src/renderer/gl_engine/tvgGlTessellator.cpp

```diff
@@ -851,12 +851,12 @@ Tessellator::~Tessellator()
 }
 
 
-bool Tessellator::tessellate(const RenderShape *rshape, bool antialias)
+bool Tessellator::tessellate(const RenderShape *rshape, const RenderPath& path, bool antialias)
 {
-    auto cmds = rshape->path.cmds.data;
-    auto cmdCnt = rshape->path.cmds.count;
-    auto pts = rshape->path.pts.data;
-    auto ptsCnt = rshape->path.pts.count;
+    auto cmds = path.cmds.data;
+    auto cmdCnt = path.cmds.count;
+    auto pts = path.pts.data;
+    auto ptsCnt = path.pts.count;
 
```

Comment:
let's go simpler:
`
this->visitShape(path->cmds.data, path->cmds.count, path->pts.data, path->pts.count);
`